README.md

```diff
@@ -73,16 +79,16 @@ summary.observe(4.7) // Observe the given value
 
 ## Info
 
-Info tracks key-value information, usually about a whole target.
+Info tracks key-value information, usually about a whole target. These are typically helpful details like git sha or other metadata.
```

Comment:
This entire section should go, it's not in the source anymore (that's my bad)

Reply:
If you don't want to include it here, I'll create a separate commit/PR to remove this.

Reply:
Great eyes, thanks!